src/stories/pages/AddExperiment/AddExperiment.js

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import ExperimentServices
+  from '../../../firebase/CRUDServices/experimentServices';
+
+import { FormItem } from '../../components/FormItem/FormItem';
+import { Form } from '../../components/Form/Form';
+import { Button } from '../../components/Button/Button';
+import { Modal } from '../../components/Modal/Modal';
+
+/**
+ * Component for AddExperiment page.
+ * @param {string} theme theme of the parent component.
+ * @param {string} buttonText text to appear on the open button.
+ * @component
+ * @return {object} (
+ *   <AddExperiment isDarkTheme={isDarkTheme}>
```

Comment:
To keep things consistent I would suggest updating the parameters in this comment to match the params above (`theme` and `buttonText`) instead of `isDarkTheme`